examples/tabs/css/tabs.css

```diff
@@ -1,107 +1,63 @@
 .tabs {
-  width: 20em;
   font-family: "lucida grande", sans-serif;
 }
 
 [role="tablist"] {
-  margin: 0 0 -0.1em;
-  overflow: visible;
+  min-width: 550px;
 }
 
-[role="tab"] {
+[role="tab"],
+[role="tab"]:focus,
+[role="tab"]:hover {
   position: relative;
+  z-index: 2;
+  top: 2px;
   margin: 0;
-  padding: 0.3em 0.5em 0.4em;
+  padding: 3px 3px 4px;
   border: 1px solid hsl(219deg 1% 72%);
-  border-radius: 0.2em 0.2em 0 0;
-  box-shadow: 0 0 0.2em hsl(219deg 1% 72%);
+  border-bottom: 2px solid hsl(219deg 1% 72%);
+  border-radius: 5px 5px 0 0;
   overflow: visible;
-  font-family: inherit;
-  font-size: inherit;
   background: hsl(220deg 20% 94%);
-}
-
-[role="tab"]:hover::before,
-[role="tab"]:focus::before,
-[role="tab"][aria-selected="true"]::before {
-  position: absolute;
-  bottom: 100%;
-  right: -1px;
-  left: -1px;
-  border-radius: 0.2em 0.2em 0 0;
-  border-top: 3px solid hsl(20deg 96% 48%);
-  content: "";
+  outline: none;
+  font-weight: bold;
 }
 
 [role="tab"][aria-selected="true"] {
-  border-radius: 0;
-  background: hsl(220deg 43% 99%);
-  outline: 0;
-}
-
-[role="tab"][aria-selected="true"]:not(:focus):not(:hover)::before {
-  border-top: 5px solid hsl(218deg 96% 48%);
-}
-
-[role="tab"][aria-selected="true"]::after {
-  position: absolute;
-  z-index: 3;
-  bottom: -1px;
-  right: 0;
-  left: 0;
-  height: 0.3em;
+  padding: 2px 2px 4px;
+  border-width: 2px;
+  border-bottom-color: hsl(220deg 43% 99%);
   background: hsl(220deg 43% 99%);
-  box-shadow: none;
-  content: "";
 }
 
```

Comment:
Unselected tabs look like they are drawn 1px below the bottom of the selected tab (or the top of the tabpanel).
I think there needs to be one more rule that says that unselected tabs only have a 1px bottom border instead of 2px:
```suggestion
[role="tab"][aria-selected="false"] {
    border-bottom: 1px solid hsl(219deg 1% 72%);
}
```

Before new rule:
![image](https://user-images.githubusercontent.com/3331913/150025555-59f02f6b-dbd8-4303-b981-c0082e270c54.png)


After new rule:
![image](https://user-images.githubusercontent.com/3331913/150025344-da0172e1-add0-41b0-bd0f-9e3cce11a8df.png)